pkg/etcd-backup-operator/controller/sync.go

```diff
@@ -15,9 +15,9 @@
 package controller
 
 import (
-	"fmt"
```

Comment:
No hurry. Let's fix all the issues in s3_backup.go first before submitting changes of this file?

Reply:
yeah, will do that.